app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Something here that I forgot to explain. I know that ConstraintLayout permits to add views, in spite of the order they have in xml. However, because of reading and accessibility purposes it's better to read it from top to bottom or left to right. Please change it.

Reply:
Moreover, there's missing some picture with a gradient that appears in the card. Please, add it to this PR.